.github/workflows/maven.yml

```diff
@@ -0,0 +1,30 @@
+# This workflow will build a Java project with Maven
+# For more information see: https://help.github.com/actions/language-and-framework-guides/building-and-testing-java-with-maven
+
+name: Java CI with Maven
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  test:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-18.04, macOS-latest, windows-2016]
+        java: [8, 8.0.192, 11, 11.0.3, 13, 13.0.4, 15, 16-ea]
+      fail-fast: false
+      max-parallel: 4           
+    name: Test JDK ${{ matrix.java }}, ${{ matrix.os }}
+
+    steps:
+    - uses: actions/checkout@v1
+    - name: Set up JDK
+      uses: actions/setup-java@v1
+      with:
+        java-version: ${{ matrix.java }}
+    - name: Test with Maven
+      run: mvn test -B --file pom.xml
```

Comment:
Testing cargo it's a bit more complicated since first we need to have an instance of Payara running. 

I'm planning to solve this as soon as the version of Cargo Java EE 7 will be closed. Until then I'll not merge merge it because the current version of Cargo has to be compatible with JDK 7

Reply:
But the workflow could also support JDK 7, if you like?

Reply:
It's fine this way. We'll don't have to worry about JDK 1.7.  I have already introduced the migration to Primefaces 8 for solving some bugs.

However, I'm still not going to merge it now because I still need to fix the testing life-cycle within maven